File: client/src/app/_models/process/deployment.ts
export interface IDeployment
{
	id: number;
	cVRefId: number;
	transactionDate: Date;
	sequence: number;
	nextSequence: number;
	nextStageDate: Date;
}


export class Deployment implements IDeployment
{
	id: number=0;
	cVRefId: number=0;
	transactionDate: Date=new Date();
	sequence: number=0;
	nextSequence: number=0;
	nextStageDate: Date=new Date();
}

File: api/Entities/Deployments/Deployment.cs
namespace api.Entities.Deployments
{
    public class Deployment: BaseEntity
    {
        public int CVRefId { get; set; }
        public DateTime TransactionDate { get; set; }   
        public int Sequence { get; set; }
        public int NextSequence { get; set; }
        public DateTime NextSequenceDate { get; set; }
    }
}